### PR TITLE
Octane is here

### PR DESCRIPTION
**Pull request by mixonic**
"Octane is Here" post from @wycats. Pairs with the 3.15 release post: https://github.com/ember-learn/ember-blog/pull/388

### CODE REVIEW

**Comment by rwjblue** [comment]
```
source/2019-12-20-octane-is-here.md
    18:245-18:249  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
      50:69-50:73  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
    50:165-50:172  warning  `special` may be insensitive, use `has a disability`, `person with a disability`, `people with disabilities` instead  special  retext-equality
    50:214-50:218  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
      58:28-58:34  warning  `easily` may be insensitive, try not to use it                                                                        easy     retext-equality
    58:104-58:108  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
     123:8-123:12  warning  `easy` may be insensitive, try not to use it                                                                          easy     retext-equality
   131:96-131:100  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
    226:21-226:25  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
  286:106-286:113  warning  `special` may be insensitive, use `has a disability`, `person with a disability`, `people with disabilities` instead  special  retext-equality
```

File: source/2019-12-20-octane-is-here.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
+---
+title: Octane is Here
+author: Yehuda Katz
+tags: Recent Posts, 2019, Announcement
+responsive: true
```
Comment (MelSumner) [inline source/2019-12-20-octane-is-here.md:6]:
```suggestion
alias: 2019-12-20-octane-is-here.md
responsive: true
```

File: source/2019-12-20-octane-is-here.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+Once you got through the tutorial, it's time to build something real for fun. The Ember addon ecosystem is a great part of Ember, so you'll want to reach for addons to speed up the process of building your project.
+
+[Ember Observer](https://emberobserver.com/) is a directory for the Ember addon ecosystem. Each addon gets a quality score based on a human review of formal criteria like the existence of a meaningful README, whether the addon has an automated build, and whether the addon is maintained by more than one person. This week, it will also indicate whether an addon is Octane Ready.
+
+Because of the details of Octane's compatibility story, most addons should be Octane Ready without any changes. Ember Observer will help the community proactively identify and fix Octane problems in maintained packages.
+
+## A Closer Look at Seamless Interop
+
+In addition to the elimination of computed properties, the Glimmer reactivity model also doesn't include special Ember proxies or observers. The Octane reactivity model is more powerful than the classic one, but it's much easier to use.
```
Comment (MelSumner) [inline source/2019-12-20-octane-is-here.md:289]:
```suggestion
<!--alex ignore special-->
In addition to the elimination of computed properties, the Glimmer reactivity model also doesn't include special Ember proxies or observers. The Octane reactivity model is more powerful than the classic one, but it's much easier to use.
```

File: source/2019-12-20-octane-is-here.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+export default EmberComponent.extend({
+  tagName: 'p',
+  classNames: ["tooltip"],
+  classNameBindings: ["isEnabled:enabled", "isActive:active"],
+})
+```
+
+#### After: Glimmer Components
+
+In contrast, Glimmer components allow you to treat the root element just like any other element. This substantially simplifies the component model, eliminating the special cases that come from having a second API just for working with the root element of a component.
```
Comment (MelSumner) [inline source/2019-12-20-octane-is-here.md:52]:
```suggestion
<!--alex ignore special-->
In contrast, Glimmer components allow you to treat the root element just like any other element. This substantially simplifies the component model, eliminating the special cases that come from having a second API just for working with the root element of a component.
```

**Comment by MelSumner** [comment]
> ```
> source/2019-12-20-octane-is-here.md
>     18:245-18:249  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
>       50:69-50:73  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
>     50:165-50:172  warning  `special` may be insensitive, use `has a disability`, `person with a disability`, `people with disabilities` instead  special  retext-equality
>     50:214-50:218  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
>       58:28-58:34  warning  `easily` may be insensitive, try not to use it                                                                        easy     retext-equality
>     58:104-58:108  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
>      123:8-123:12  warning  `easy` may be insensitive, try not to use it                                                                          easy     retext-equality
>    131:96-131:100  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
>     226:21-226:25  warning  `just` may be insensitive, try not to use it                                                                          just     retext-equality
>   286:106-286:113  warning  `special` may be insensitive, use `has a disability`, `person with a disability`, `people with disabilities` instead  special  retext-equality
> ```

I added alex ignores for "special" but I think we should try to remove some of these "easy" and "just" references.

File: source/2019-12-20-octane-is-here.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+export default EmberComponent.extend(Tabs, {
+  // ...
+});
+```
+
+The drawbacks of using mixins for UI composition are well-described [across](https://reactjs.org/blog/2016/07/13/mixins-considered-harmful.html) [the](https://blog.logrocket.com/hooks-are-coming-to-vue/) [JavaScript](https://github.com/sveltejs/svelte/issues/1041#issuecomment-353660981) [ecosystem](https://github.com/emberjs/rfcs/issues/534). The most glaring issue is naming conflicts. Any method on a mixin might conflict with a method on any other mixin, with no good way to resolve the conflicts.
+
+In the context of Ember, there's another issue with using Ember Component mixins for reusable DOM behavior. If you want to use the `Tabs` mixin on an element, you need to turn that element into a component with a JavaScript class, which is pretty awkward.
+
+You can still use mixins in Ember 3.15, and addons may still provide mixins for you to use.
```
Comment (MelSumner) [inline source/2019-12-20-octane-is-here.md]:
```suggestion
While we do recommend you avoid mixins, you can still use them in Ember 3.15. Addons may also still provide mixins for you to use.
```

**Comment by mixonic** [comment]
Thanks for the cleanup! @MelSumner @rwjblue I'm going fold these commits into https://github.com/ember-learn/ember-blog/pull/388

**Pull request closed without merging**